### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by liuzzi** [cheng6076/NeuralSum#1]
Hello, I have run your code and I am wondering if I am reaching what you observed in your paper.
It seems that by epoch 3 most of the training has converged, to its lowest it will in all of the training.
Around epoch 14 the training stopped because the learning rate decreased so low and I witnessed val loss as:

`
        > validation loss = 0.71172339, perplexity = 2.03749967
        > validation loss = 0.66335487, perplexity = 1.94129419
        > validation loss = 0.63958973, perplexity = 1.89570296
        > validation loss = 0.69881624, perplexity = 2.01137042
        > validation loss = 0.71748960, perplexity = 2.04928231
        > validation loss = 0.70213819, perplexity = 2.01806307
        > validation loss = 0.72103143, perplexity = 2.05655336
        > validation loss = 0.65930551, perplexity = 1.93344915
        > validation loss = 0.72959179, perplexity = 2.07423377
        > validation loss = 0.69812429, perplexity = 2.00997901
        > validation loss = 0.68849707, perplexity = 1.99072134
        > validation loss = 0.58283472, perplexity = 1.79110849
        > validation loss = 0.70136720, perplexity = 2.01650786
        > validation loss = 0.67284966, perplexity = 1.95981419
        > validation loss = 0.74536568, perplexity = 2.10721183
        > validation loss = 0.65719879, perplexity = 1.92938018
        > validation loss = 0.70397091, perplexity = 2.02176499
        > validation loss = 0.69068134, perplexity = 1.99507439
        > validation loss = 0.67542088, perplexity = 1.96485972
`

I did not use pretrain (what is pretrain for, anyway?)   I just started immediately with the training.

Eval results look better than random chance, however I feel like I am missing something here, you dont mention anything about loss or perplexity in your paper, so I am uncertain if I am reproducing your results.
Thank you

**Comment by suryanshagnihotri** [comment]
![image](https://user-images.githubusercontent.com/8201334/31646486-b0984218-b31f-11e7-8ac6-96d3ea234b45.png)
did you get error like this while training the model (running python train.py)
